templates/ea_rule.yaml.j2

```diff
@@ -14,7 +14,17 @@ timeframe:
 num_events: {{num_events}}
 {% endif %}
 
-filter: {{ filter }}
+{% if filter is defined %}
```

Comment:
Why not use just  {{ filter }}?

Reply:
In general case this property is optional.

Reply:
```
{% if filter is defined %}
filter: {{ filter | to_yaml }}
{% endif %}